lib/topological_inventory/orchestrator/source_type.rb

```diff
@@ -6,6 +6,14 @@ class SourceType < ApiObject
       SUPPORTED_TYPES = %w[amazon ansible-tower azure openshift].freeze
       AVAILABILITY_CHECK_SOURCE_TYPES = %w[amazon ansible-tower openshift].freeze
 
+      def method_missing(name, *args, &block)
+        if name.to_s.match?(/is_.*\?/)
```

Comment:
@lindgrenj6 I like this, but with https://rubocop.readthedocs.io/en/latest/cops_naming/ I suggest going back to `azure?` instead of `is_azure?`

Reply:
Ok, switched back to the old style _but_ it is all handled in `method_missing` now instead of meta-methods.